source/schemas/tracking_links.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Tracking Link",
+  "type": "object",
+  "description": "The tracking link resource is the data used to construct the tracking link from the token, mapping a short link for a job back to a referrer and source.",
+  "properties": {
+    "id": {
+      "description": "The id of the tracking link.",
+      "title": "ID",
+      "type": "integer"
+    },
+    "job_id": {
+      "description": "The id of the job associated with this tracking link.",
+      "title": "Job ID",
+      "type": "integer"
+    },
+    "token": {
+      "description": "The token included in the job board link.",
+      "title": "Token",
+      "type": "string"
+    },
+    "related_post_id": {
+      "description": "The ID of the related post or email for this job token",
+      "title": "Related Post ID",
+      "type": ["integer", "null"]
```

Comment:
Because it describes a type, not a value. If the value were `null` it could get stripped out